src/templates/pycontw-2020/contents/en/speaking/tutorial.html

```diff
@@ -0,0 +1,47 @@
+{% extends 'contents/_base.html' %}
+
+{% block title %}How to Propose a Tutorial{% endblock title %}
+
+{% block content %}
+{% url 'page' path='speaking/cfp' as speaking_cfp_url %}
+{% url 'page' path='speaking/talk' as speaking_talk_url %}
+
+<h1>How to Propose a Tutorial?</h1>
+
+<p>For general proposal calling information, see <a href="{{ speaking_cfp_url }}">Call for Proposals</a>.</p>
+
+<p>First of all, thank you considering giving a tutorial in PyCon Taiwan 2020.</p>
+
+<p>Generally speaking, Good tutorials possess most of the traits good talks have. The most discernible differences of a tutorial from a <a href="{{ speaking_talk_url }}">talk</a> are the <em>hands-on</em> nature and the <em>longer duration</em>, and it's <em>paid</em>. By longer duration, generally more content is required and it should remain interesting and keep the attendees awake and hyped even after hours of lecture and exercises. By containing hands-on exercises, experience of teaching coding to people is necessary. You might also need teaching assistants who you should be able to comfortably communicate during the tutorial, usually at least two teaching assistants for a general class size of 20 people is needed.</p>
```

Comment:
This year, tutorials are free.

Reply:
update the sentence by removing "it's paid."